python/ray/data/_internal/logging.py

```diff
@@ -11,6 +11,12 @@
     os.path.join(os.path.dirname(__file__), "logging.yaml")
 )
 
+# Env. variable to specify the encoding of the file logs when using the default config.
+RAY_DATA_LOG_ENCODING = "RAY_DATA_LOG_ENCODING"
+
+# Env. variable to specify the logging config path use defaults if not set
+RAY_DATA_LOGGING_CONFIG_PATH = "RAY_DATA_LOGGING_CONFIG_PATH"
```

Comment:
Nit: Maybe rename to make it clear these refer to environment variable names and not the values? I'd think `RAY_DATA_LOGGING_CONFIG_PATH` refers to the actual path and not the name of an environment variable.